### DISCUSSION TITLE
Ping Discord Membergroup(s)

### DISCUSSION THREAD

**Discussion by ThistleProse** [RevertIT/mybb-rt_discord_webhooks#5] [Ideas]
Hello,

I would like to be able to ping selected usergroups of the Discord Server when the RT makes a bot post. Eg;
![image](https://github.com/RevertIT/mybb-rt_discord_webhooks/assets/28851048/d1a5cdc0-d663-4106-8341-f6957a939b02)

This is the result of the bot I am currently using. I am happy with it's results, but it is less clean on the admin panel compared to yours :)

This is the inside of the settings for that result; it is pushed depending on the Prefix of the topic;
![image](https://github.com/RevertIT/mybb-rt_discord_webhooks/assets/28851048/cd5a3842-68af-4dca-b8bf-e8a378765394)
![image](https://github.com/RevertIT/mybb-rt_discord_webhooks/assets/28851048/b9decf48-65c9-4d5c-9ebc-14517224d23d)

Is there any way something like this could be implemented into your Plugin?

Thank you for your time!

**Comment by RevertIT** [comment]
I guess we can do that, should be relatively easy.

Is the format of the role to ping `<@&XX>`?

Edit: https://github.com/RevertIT/mybb-rt_discord_webhooks/releases/tag/v1.5

**Reply by ThistleProse** [reply]
Yes, it is. You can find individual group ids in the developer mode. I can't remember how off the top of my head, and I'm on my phone at the moment, but it's that, yes.

**Reply by RevertIT** [reply]
Alright, when you have time test it out, so we can push it to the MyBB site.

**Reply by ThistleProse** [reply]
Hey! Replied below in a new comment or something XD Github threads are a bit odd to me lol

**Reply by RevertIT** [reply]
You just enable the embeds and mentions for webhook, then you add in post <@&id> and it will ping the group, same as @here and @everyone

**Comment by ThistleProse** [comment]
Hello!

Thanks for your swift response! It's appreciated :)

Unfortunately I'm not sure what changes were made? I found this setting has been added:
![image](https://github.com/RevertIT/mybb-rt_discord_webhooks/assets/28851048/675fc73e-b65f-40a2-b496-47cafddaa3be)

Or I guess, more accurately, I'm not sure how to make the most of this new addition?

Thank you!
